templates/_composition/CaliburnMicro/Feature.ShareTargetExamplePage/ViewModels/wts.ItemNameExampleViewModel.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Threading.Tasks;
+using System.Windows.Input;
+using Windows.ApplicationModel.DataTransfer;
+using Windows.ApplicationModel.DataTransfer.ShareTarget;
+using Windows.Storage;
+using Windows.UI.Xaml.Media.Imaging;
+using Caliburn.Micro;
+using Param_ItemNamespace.Helpers;
+using Param_ItemNamespace.Models;
+
+namespace Param_ItemNamespace.ViewModels
+{
+    public class wts.ItemNameExampleViewModel : Screen
+    {
+        private ShareOperation _shareOperation;
+
+        private string _pageTitle;
+
+        public string PageTitle
+        {
+            get => _pageTitle;
+            set => Set(ref _pageTitle, value);
+        }
+
+        private SharedContentModel _sharedData;
+
+        public SharedContentModel SharedData
+        {
+            get => _sharedData;
+            set => Set(ref _sharedData, value);
+        }
+
+        public wts.ItemNameExampleViewModel()
+        {
+        }
+
+        public async Task LoadAsync(ShareOperation shareOperation)
```

Comment:
You can use the `OnInitialize` method on the view model to call this to wire it into the view model life cycle.